### ISSUE TITLE
Quick Blox Blocking Complete Private and Public Group

### ISSUE THREAD

**Issue by umairjamalian** [QuickBlox/quickblox-ios-sdk#525]
I m using this framework for chat functionality, i was trying to add block user functionality in private chat and had successfully completed that.

Now i m facing issue when i try to block a complete group whether its private or public group. I don't know which id to add in privacy list.

Kindly guide me on this issue if anyone knows about it.

**Comment by VitaliyGuru** [comment]
Hi @umairjamalian , You can create privacy item like this, e.g. 
`QBPrivacyItem *item = [[QBPrivacyItem alloc] initWithType:type valueForType:valueType action:DENY];`
where `type` will be:
-  USER_ID if u want to block a user in 1-1 chat 
-  GROUP_USER_ID if u want to block in a group chat.

**Comment by umairjamalian** [comment]
Hi @VitaliyGuru i have done single user blocking which is private chat.

My question is that like for blocking private chat we use occupant ID in value for type parameter and it works, But what ID to pass in ITEM for group?

If we block all users of group then it also block the private chat of that user who exist in group chat. What is the for this problem.

Is there any way to block a group notifications instead of user in a group?

**Comment by VitaliyGuru** [comment]
@umairjamalian 
Now we're investigating XMPP documentation. 
We will answer you ASAP.

**Comment by umairjamalian** [comment]
Okay waiting for your response

**Comment by umairjamalian** [comment]
@VitaliyGuru
Any luck ?

**Comment by Raikerian** [comment]
@umairjamalian there is no such feature yet and no plans to implement it in a near future. For now you can block users, or block chats by creating a [custom object](http://quickblox.com/developers/SimpleSample-customObjects-ios) of blocked chats, to not receive messages from it.

To stay in touch with our updates follow our [blog](http://quickblox.com/blog).

**Comment by hiteshtrivedi** [comment]
Hi
I am faceing issue to block private chat and unblock it. Can you please help me.
is it possible by QBRequest?

**Comment by Raikerian** [comment]
Hi @hiteshtrivedi 

No, it is only possible through privacy lists natively. You can of course build your own blocking system using Custom objects if you need only QBRequest's. Also create a separate issue if this does not answer your question.
